wallet/storage/src/lib.rs

```diff
@@ -69,8 +72,12 @@ pub trait WalletStorageRead {
 pub trait WalletStorageWrite: WalletStorageRead {
     /// Set storage version
     fn set_storage_version(&mut self, version: u32) -> Result<()>;
-    fn set_utxo(&mut self, outpoint: &AccountOutPointId, entry: Utxo) -> Result<()>;
-    fn del_utxo(&mut self, outpoint: &AccountOutPointId) -> Result<()>;
+    fn set_block(
+        &mut self,
+        block_height: &AccountBlockHeight,
+        block: &WalletBlock,
+    ) -> crate::Result<()>;
```

Comment:
Why do we need to store the whole block in the wallet?!!!! That's crazy and should be avoided at all cost as it can blow up the db storage size! This information should strictly be stored in the node and retrieved from the node.